modules/ebi-metagenomics/bwamem2decontnobams/main.nf

```diff
@@ -0,0 +1,55 @@
+process BWAMEM2DECONTNOBAMS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' :
+        'biocontainers/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' }"
+
+
+    input:
+    tuple val(meta), path(reads)
+    tuple val(meta2), path(index)
```

Comment:
This isn't a request for a change, you can keep it this way but just starting a conversation about something...

`meta2` Is this the standard for when you need more than one meta? I think the official definition of `meta` in nf-core is metadata about samples, and `meta2` here is the reference genome.. so I wonder whether it makes more sense to be more explicit when we include more than one meta-type input

Reply:
Do we need a meta here at all? If that is index of reference genome we know all info from filename and location (name, version, organism, etc). What do we store in meta2?

Reply:
Yes, the idea here is to track the different reference genomes we are using in each round of decontamination. There's a discussion about using 'meta' and 'meta2', and this form is the standard at the moment for [Best practices for multiple process inputs](https://github.com/nf-core/modules/issues/4311)

Reply:
> There's a discussion about using 'meta' and 'meta2', and this form is the standard at the moment for [Best practices for multiple process inputs](https://github.com/nf-core/modules/issues/4311)

sounds good to me then, keep it as is for sure!